dbld/images/helpers/pip_packages.manifest

```diff
@@ -1,5 +1,4 @@
 # pip packages for successful run unit tests
-hy                      [centos, fedora, debian, ubuntu]
```

Comment:
Hy is used for internal testing, so the package is needed. Though we do not test it on centos6. Is it possible to exclude only from centos6?